oca_event_badge/README.rst

```diff
@@ -0,0 +1,51 @@
+===================
+OCA Custom Settings
+===================
+
+This module allow to add specific tags for oca partners.
+These tags are printed on the badge attendee.
```

Comment:
Please use the new readme fragments: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme